docs/contributors.md

```diff
@@ -0,0 +1,56 @@
+sidebar_label: Contributors
+---
+
+## Anson Lau
+
+Anson is a technical educator at Web3 Foundation. Before that, he worked as a Hyperledger blockchain
+developer to deliver B2B application at IBM. He is passionate about staking, governance,
+cryptoeconomics, and privacy areas. Anson has been actively participating in the blockchain space
+since 2017 and creating video content to help different people to understand how decentralization
+could deliver us a better future.
+
+## Bill Laboon
+
+Bill Laboon is the Technical Education Lead at the Web3 Foundation, after spending the previous five
+years teaching Computer Science at the University of Pittsburgh. He is the author of two books: A
+Friendly Introduction to Software Testing, an undergraduate textbook; and Strength in Numbers, a
+near-future novel set in a world in which cryptocurrency has eliminated traditional money.
+
+## Bruno Skvorc
+
+Bruno is a blockchain developer and educator from Croatia. He joined the ecosystem full time in 2015
+with the advent of Ethereum, and created educational resources, programming courses, tutorials and
+newsletters for budding developers of the industry. Since 2018 he had been working on one of the
+teams building the next generation of Ethereum, and since 2019 he's been a technical educator at
+Web3 Foundation, dedicated to forwarding the web3 vision and helping make sure everyone, everywhere
+has access to the Internet of Value and Trust. Bruno is passionate about decentralization,
+censorship resistance, and transparency coupled with accountability, so he's found his passion in
+immutable ledgers of the present and future. He holds masters' degrees in Computer Science and
+English Language and Literature and likes to dabble in virtual reality when possible. He can be
+found on Twitter at @bitfalls.
+
+## Joe Petrowski
+
+Joe is a research analyst at Parity Technologies, where he focuses on making Substrate chain
+integration as smooth as possible, from educating users on Substrate and Polkadot fundamentals to
+building tools to create great user experiences and occasionally writing a bit of code. He also
+hosts the [Relay Chain](https://relaychain.fm) podcast. Prior to working at Parity, Joe worked in
+shock and vibration simulation for satellite launch, algorithmic trading, and professional cycling.
+He has a Bachelor of Science in Aeronautical and Mechanical Engineering.
+
+## Logan Saether
+
+Logan is a Technical Educator at Web3 Foundation where he works on creating material for supporting
+the development of the next generation of distributed technologies. At Web3 Technical Education,
+Logan focuses on creating the Polkadot Wiki as the central source of truth and knowledge for the
+Polkadot protocol. He contributes to the creation of written content and example applications that
+demonstrate how to use these new technologies.
+
+## Kirsten Richard
+
+Kirsten is a Technical Educator intern at the Web3 Foundation from Canada. She's passionate about
```

Comment:
```suggestion
Kirsten is a Technical Educator intern at Web3 Foundation from Canada. She's passionate about
```

Reply:
Why not https://github.com/w3f/polkadot-wiki/pull/878/files#diff-9558ae512b5cea68efed66b2b4b0b233R17 ?

Reply:
Because I'm a hypocrite, apparently.